### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by Jmeyer1292**
This PR:
1. Adds a line to the readme instructing users to install libqd
2. Adds a description of how to execute the generated trajectory bag files in this version of Godel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   sudo apt-get install python-wstool
   ```
 
+- Install QD either from [source](http://crd-legacy.lbl.gov/~dhbailey/mpdist/) or, on Ubuntu 14.04 or later, through apt-get:
+ ```
+ sudo apt-get install libqd-dev
+ ```
+
```
Comment (jrgnicho) [inline README.md]: I notice that octomap-ros also needs to be installed.

**Comment by gavanderhoorn** [comment]
@Jmeyer1292 wrote:

> 1 . Adds a line to the readme instructing users to install libqd

`libqd-dev` is a dependency of `godel_openvoronoi` ([here](https://github.com/ros-industrial-consortium/godel_openvoronoi/blob/549f3802c9c3cfbd2e43c01e5e44c2966c713901/src/package.xml#L17)), and should be installed automatically when running the `rosdep install ..` command mentioned in the readme. A manual `apt-get install` should not be necessary.

Did you encounter issues when following the install procedure as described in the current version of the readme?

File: README.md (hunk 1 above)
Comment (gavanderhoorn) [inline README.md]: Shouldn't this also be installed by `rosdep install ..`? It is declared a `run` and `build_depend` of `godel_surface_detection` ([here](https://github.com/ros-industrial-consortium/godel/blob/78ee0f4cb757035114f759110e7d5d69159a1b96/godel_surface_detection/package.xml#L15))?

**Comment by Jmeyer1292** [comment]
Now it's hard to be 100% certain, but if I remove libqd and try to install from scratch, I do not get qd and I do not get octomap ros. This is following the existing instructions. I believe Jorge is running on a fresh VM too.

I'm probably just overlooking something.

**Comment by Jmeyer1292** [comment]
Going back through I realized that rosdep was complaining about not having build keys for `motoman_sia20d_support`, and just stopped. If I clone motoman into the workspace, it does indeed install libqd-dev.

**Comment by Jmeyer1292** [comment]
And I was overlooking something. On my machine, rosdep was complaining about missing keys for `motoman_sia20d_support` and was failing before it installed any dependencies. 

@jrgnicho  can you describe your issues?

**Comment by jrgnicho** [comment]
For me it also complains about the **motoman_sia20d_support** dependency, the right fix is to list the _motoman_ metapackage as a dependency.

**Comment by gavanderhoorn** [comment]
@Jmeyer1292 wrote:

> On my machine, rosdep was complaining about missing keys for `motoman_sia20d_support` and was failing before it installed any dependencies. 

Which is to be expected, as none of the `motoman` packages have been released into Indigo (see [debbuild/indigo?q=motoman](http://www.ros.org/debbuild/indigo?q=motoman) fi).

@jrgnicho wrote:

> For me it also complains about the **motoman_sia20d_support** dependency, the right fix is to list the _motoman_ metapackage as a dependency.

No, metapackages are not supposed to be used for dependency declarations.

The fix would be to update the `.rosinstall` file to include the `motoman` repository. At least until `motoman_sia20d_support` has been released.

This has actually already been reported in #53, #58 and #62 (although `industrial_core` pkgs have been released since then).

**Comment by gavanderhoorn** [comment]
For getting `rosdep` to continue after encountering errors with specific keys, use the `-r` switch.

**Comment by Jmeyer1292** [comment]
@gavanderhoorn Thank you for the clarification. I didn't realize we didn't have an indigo release of Motoman.

I removed the install-related instructions from this PR, leaving only the run instructions. We will create an indigo-devel branch of Godel and move forward in that branch.

**Comment by jrgnicho** [comment]
+1, We'll need to add the motoman source dependency in the godel.rosinstall file for the indigo branch.

**Comment by Jmeyer1292** [comment]
Merging; Thank you for reviewing.

**Pull request merged**